packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -124,6 +132,16 @@ export class Subscribers extends WithHttp implements ISubscribers {
     );
   }
 
+  async markAllMessagesAs(
+    subscriberId: string,
+    markAs: IMarkAllMessageAs
```

Comment:
feedId query param is missing 🙂

Reply:
I think we should create a new interface with`markAs` and `feedIdentifier` fields